README.dev.md

```diff
@@ -57,8 +57,7 @@ To contribute to TulipaEnergyModel.jl, you need the following:
     julia
     ```
 
-    > **Note**:
-    > `julia` must be part of your environment variables to call it from the
+    > **Note:** > `julia` must be part of your environment variables to call it from the
```

Comment:
```suggestion
    > **Note:**
    > `julia` must be part of your environment variables to call it from the
```
Looking at the other lines, I think you didn't mean to join these.

Reply:
Autoformat on save is changing it. :/